packages/ipns/src/index.ts

```diff
@@ -379,6 +380,13 @@ export interface RepublishOptions extends AbortOptions, ProgressOptions<Republis
   interval?: number
 }
 
+export interface RepublishRecordOptions extends AbortOptions, ProgressOptions<RepublishProgressEvents | IPNSRoutingEvents> {
+  /**
+   * Only publish to a local datastore (default: false)
```

Comment:
Tells tsdoc what the default value is:

```suggestion
   * Only publish to a local datastore
   *
   * @default false
```